master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -265,11 +265,11 @@ private[celeborn] class Master(
   }
 
   masterSource.addGauge(MasterSource.DEVICE_CELEBORN_TOTAL_CAPACITY) { () =>
-    statusSystem.workers.asScala.toList.map(_.totalSpace()).sum
+    statusSystem.getAvailableWorkers.asScala.toList.map(_.totalSpace()).sum
   }
 
   masterSource.addGauge(MasterSource.DEVICE_CELEBORN_FREE_CAPACITY) { () =>
-    statusSystem.workers.asScala.toList.map(_.totalActualUsableSpace()).sum
+    statusSystem.getAvailableWorkers.asScala.toList.map(_.totalActualUsableSpace()).sum
```

Comment:
maybe only getAvailableWorkers for `DEVICE_CELEBORN_FREE_CAPACITY`.

And continue to use `workers` for `DEVICE_CELEBORN_TOTAL_CAPACITY`.